agent/src/main/resources/build-scan-init.gradle

```diff
@@ -21,21 +53,49 @@ def buildScanPublishedAction = { def buildScan ->
 if (GradleVersion.current() < GradleVersion.version('6.0')) {
     // Gradle 4.1+ and 5.+
     rootProject {
+        if (autoApplyGe && !pluginManager.hasPlugin(BUILD_SCAN_PLUGIN_ID)) {
+            pluginManager.apply(initscript.classLoader.loadClass(BUILD_SCAN_PLUGIN_CLASS))
+        }
+
+        if (autoApplyCcud && !pluginManager.hasPlugin(CCUD_PLUGIN_ID)) {
+            pluginManager.apply(initscript.classLoader.loadClass(CCUD_PLUGIN_CLASS))
+        }
+
         pluginManager.withPlugin(BUILD_SCAN_PLUGIN_ID) {
             buildScanPublishedAction(buildScan)
+            buildScan {
+                afterEvaluate {
```

Comment:
Having afterEvaluate nested inside buildScan seems a bit odd.

Reply:
I think I moved move this entire afterEvaluate block to within the if statement above (autoApplyCcud && !pluginManager.hasPlugin(CCUD_PLUGIN_ID) - and wrap it again with `pluginManager.withPlugin(BUILD_SCAN_PLUGIN_ID)`

Reply:
Did you mean in the `if(autoApplyGePlugin...` block? That's what I did, since it seemed to make most sense to configure GE is we applied the GE plugin, not CCUD.